lib/mongo/query_cache.rb

```diff
@@ -84,6 +84,24 @@ def clear_cache
         Thread.current["[mongo]:query_cache"] = nil
       end
 
+      # Clear the section of the query cache storing cursors with results
+      # from this namespace.
+      #
+      # @param [ String ] namespace The namespace to be cleared, in the format
+      #   "database.collection".
+      #
+      # @return [ nil ] Always nil.
+      #
+      # @api private
+      def clear_namespace(namespace)
+        cache_table[namespace] = nil if cache_table[namespace]
```

Comment:
I think this and the next call can be simplified to `cache_table.delete(namespace)`.

Reply:
Thank you!